scripts/generator/graphql_generator/csharp/SDK/Serializable.cs.erb

```diff
@@ -4,10 +4,11 @@ namespace <%= namespace %>.SDK {
     using System.Collections;
     using System.Collections.Generic;
     using UnityEngine;
+    using <%= namespace %>.MiniJSON;
 
     public class Serializable {
         public string ToJsonString() {
-            return JsonUtility.ToJson(this);
+            return Json.Serialize(this);
```

Comment:
If that's the case then it doesn't make sense for there to be a `Serializable` subclass anymore since we don't want to implement `IDictionary`/`IList` here. It makes more sense to 🔥 this class and have `ApplePayEventResponse` implement `IDictionary` and replace the `ToJsonString()` calls to just `Json.Serialize(applePayEventResponseInstance)`.